include/pressiodemoapps/swe2d.hpp

```diff
@@ -94,16 +94,11 @@ RetType
  InviscidFluxReconstruction inviscRecEn)
 {
 
-  if (problemEnum == Swe2d::SlipWall)
-  {
-    return RetType(implswe2d::TagProblemSlipWall{},
-		   meshObj, inviscRecEn,
+  if (problemEnum == Swe2d::SlipWall){
+    return RetType(implswe2d::TagProblemSlipWall{}, meshObj, inviscRecEn,
 		   ::pressiodemoapps::InviscidFluxScheme::Rusanov,
-		   9.8,  // gravity
-		   -3.0, // coriolis
-		   0.125); // pulse mag
+		   implswe2d::create_vec_with_default_params<typename RetType::scalar_type>());
   }
```

Comment:
Would it be possible to have a paramMap as an optional argument here? Similar to how it's done for the overload with `CustomBCsFunctor`s, where it would fall back to the default values if nothing is provided, but allow specifying the parameterization without also having to specify the BC functors. Just thinking of the situation of generating monolithic training data without the Schwarz framework, where the BC functor isn't specified.